package.json

```diff
@@ -15,9 +15,9 @@
     "doc": "docs"
   },
   "scripts": {
-    "test": "node_modules/.bin/jest src/**/*.test.ts --coverage",
-    "test:ci": "node_modules/.bin/jest src/**/*.test.ts --coverage --runInBand --ci",
-    "test:clean": "node_modules/.bin/jest src/**/*.test.ts --coverage && npm run prettier:check",
+    "test": "node_modules/.bin/jest src/**/*.test.ts src/*.test.ts --coverage",
```

Comment:
why add the second glob pattern here?

Reply:
Not sure if this is only a Linux thing: The glob pattern `src/**/*.test.ts` seems to not be applied recursively but rather as `src/*/*.test.ts`. That means that the test `src/jestTensorMatchers.test.ts` is not executed. Maybe there is a better solution than adding a second glob pattern?

Reply:
I Cannot think of a simpler/cleaner solution for now. Maybe it would be better to create a `runTests.js` script and use the [glob](https://www.npmjs.com/package/glob) package to discover the files before running jest. But that seems out of scope for this PR. The workaround will have to do for now.